src/main/java/entity/room/BlackjackRoom.java

```diff
@@ -0,0 +1,52 @@
+package entity.room;
+
+import java.io.IOException;
+import java.util.List;
+
+import data_access.ApiDataAccess;
+import entity.Deck;
+import entity.player.Player;
+import use_case.ApiDataAccessInterface;
+
+/**
+ * A Blackjack-implementation of the Room interface.
+ */
+public class BlackjackRoom implements Room {
+
+    private final String roomName;
+    private final String gameName;
+    private List<Player> curPlayers;
+    private String curTurnPlayer;
+    private String status;
+    private Deck deck;
+    // TODO: should also contain an attribute to store the bet (and update the constructor).
+
+    public BlackjackRoom(String roomName, String gameName, List<Player> curPlayers) throws IOException {
+        this.roomName = roomName;
+        this.gameName = gameName;
+        this.curPlayers = curPlayers;
+        this.curTurnPlayer = curPlayers.get(0).getPlayerID();
+        this.status = "Initializing";
+        final ApiDataAccessInterface apiDataAccessInterface = new ApiDataAccess();
```

Comment:
I don't think entities are allowed to access DAIs. Maybe there can be a setter method that'll be called by a use case later on.

Reply:
This ApiDataAccessInterface is in the use_case package